### ISSUE TITLE
Build fails with vdr 2.7.1

### ISSUE THREAD

**Issue by heitbaum** [MarkusEh/vdr-plugin-live#61]
GetEvent has been deprecated
- ref:
- http://git.tvdr.de/?p=vdr.git;a=commitdiff;h=8d82b050712de0db83ac3b28d04a59479b024622;hp=930e3b4200e0bdf46264ef663e6121264ef6e2e4

```

BUILD      vdr-plugin-live (target)
    TOOLCHAIN      manual
CC pages/ edit_timer.o
edit_timer.ecpp: In member function 'virtual unsigned int {anonymous}::_component_::operator()(tnt::HttpRequest&, tnt::HttpReply&, tnt::QueryParams&)':
edit_timer.ecpp:128:57: error: 'const class cSchedule' has no member named 'GetEvent'; did you mean 'DelEvent'?
  128 |                         const cEvent *event = schedule->GetEvent( eventid );
      |                                                         ^~~~~~~~
      |                                                         DelEvent
compilation terminated due to -Wfatal-errors.
make[1]: *** [Makefile:37: edit_timer.o] Error 1
make: *** [Makefile:191: pages] Error 2
```

**Comment by MarkusEh** [comment]
Solved with 3.3.6

**Comment by heitbaum** [comment]
> Solved with 3.3.6

Thanks. Tested and builds fine now.
